### ISSUE TITLE
[Feature request] Use tracker output to set update_fields on Model.save()

### ISSUE THREAD

**Issue by arski** [carljm/django-model-utils#245]
Kind of what https://github.com/karanlyons/django-save-the-change does but using django-model-utils' tracker instead of their custom built.

The main idea is that Django's Model.save() saves all fields of the current instance, and a way to circumvent that is using the update_fields param. However, setting that manually can be annoying and error-prone, so I'm wondering if anyone has considered using something like Model.save(update_fields=tracker.changed().keys()) - this would be pretty damn awesome imo!

Cheers

**Comment by romgar** [comment]
Hi @arski !

You would imagine something like `my_model_instance.save_changed_fields()`, right?

There is a django app named `django-dirtyfields` (http://django-dirtyfields.readthedocs.io/en/develop/) that has a similar function named `save_dirty_fields`, which will call under the hood an `update` on all fields that need to be updated.

But do you want this functionality to update only the ones that are tracked through the `FieldTracker`?

**Comment by arski** [comment]
@romgar indeed, that sounds exactly like what I'd like to have. And to replace the default save() in my models with the save_dirty_fields - but that I can do ;)

**Comment by romgar** [comment]
> But do you want this functionality to update only the ones that are tracked through the FieldTracker?

Not sure if your "indeed" was for this statement at the end.

**Comment by romgar** [comment]
@arski I'm going through the backlog of issues/MR, and deal with new features probably after :-) Feel free to propose something at the meantime!
